src/cdomains/intDomain.ml

```diff
@@ -959,8 +935,7 @@ struct
   let show (x: Ints_t.t) = if (Ints_t.to_int64 x) = GU.inthack then "*" else Ints_t.to_string x
 
   include Std (struct type nonrec t = t let name = name let top_of = top_of let bot_of = bot_of let show = show let equal = equal end)
-  (* FIXME: poly compare *)
```

Comment:
Is this not still a usage of polymorphic compare?

Reply:
It doesn't appear so, because `Std` doesn't implement `compare`. I guess it must've been removed at some point, but the comment remained. And here the polymorphic `hash` is also removed from `Std`.